gomint-server/src/main/java/io/gomint/server/world/WorldAdapter.java

```diff
@@ -663,9 +663,6 @@ private void tickLoop() {
 
     @Override
     public void update(long currentTimeMS, float dT) {
-        for (io.gomint.server.entity.EntityPlayer player : this.players.keySet()) {
-            player.update(currentTimeMS, dT);
```

Comment:
Who does update the players now?

Reply:
The entity manager does this already, just like now.